bob/src/hw_metrics_collector.rs

```diff
@@ -24,20 +24,30 @@ impl HWMetricsCollector {
     }
 
     fn collect_used_disks(disks: &[DiskPath]) -> HashMap<PathBuf, String> {
-        System::new_all()
+        System::new_with_specifics(RefreshKind::new().with_disks_list().with_disks())
             .disks()
             .iter()
             .filter_map(|d| {
                 let path = d.mount_point();
+                let path_metadata = match path.metadata() {
+                    Ok(meta) => meta,
+                    Err(err) => {
+                        warn!("Can't get metadata from OS for disk '{}' returned by 'sysinfo': {:?}", path.display(), err);
```

Comment:
How often this message will be displayed? Shouldn't it be debug?

Reply:
Once for each problematic disk. `collect_used_disks` called only once at startup